frontend/Data.js

```diff
@@ -101,4 +101,21 @@ export default class Data {
       return e.response.data.errors
     }
   }
+  async createComment(comment) {
```

Comment:
this is again an amazing technique because in this case, there is only one place to adjust request method, URL, headers and API stuff.  I presume this file is responsible for handling anything related to API. Very cool ! I should definitely do it like that next time.